rest/attachment_test.go

```diff
@@ -2106,13 +2083,12 @@ func TestBasicAttachmentRemoval(t *testing.T) {
 	rt.Run("doc with multiple attachments and removal of a single one upon document update", func(t *testing.T) {
 		// Create a document.
 		docID := "foo12"
-		revID := rt.CreateDoc(t, docID)
-		require.Equal(t, "1-45ca73d819d5b1c9b8eea95290e79004", revID)
+		version := rt.CreateTestDoc(docID)
 
 		// Add an attachment to the document.
 		att1Name := "alice.txt"
 		att1Body := "this is the body of attachment alice.txt"
-		revID = storeAttachment(t, docID, revID, att1Name, att1Body)
+		revID := storeAttachment(t, docID, version.RevID, att1Name, att1Body)
```

Comment:
Seems like storeAttachment should return DocVersion, and then use UpdateDoc() on line 2152 below.